File: src/test/java/murphy/software/games/platform2d/move/frame/FrameTest.java
package murphy.software.games.platform2d.move.frame;

public class FrameTest {

    // @Test
    void testOpenFrame() throws InterruptedException {
        TestingFrame tf = new TestingFrame();
        tf.drawDot(100,100);
        Thread.sleep(2000);
        tf.drawDot(120,120);
        Thread.sleep(2000);
        tf.clear();
        tf.drawDot(150,150);
        Thread.sleep(10000);
    }
}

File: src/test/java/murphy/software/games/platform2d/move/frame/FrameTest.kt
package murphy.software.games.platform2d.move.frame

import org.junit.jupiter.api.Test

class FrameTest {

    @Test
    @Throws(InterruptedException::class)
    fun testOpenFrame() {
        val tf = TestingFrame()
        tf.drawDot(100, 100)
        Thread.sleep(1000)
        tf.drawDot(120, 120)
        Thread.sleep(1000)
        tf.clear()
        tf.drawDot(150, 150)
        Thread.sleep(2000)
    }
}